### ISSUE TITLE
Streptococcus pneumoniae growth in sputum culture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10989]
**Preferred term label:**
Streptococcus pneumoniae growth in sputum culture

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Bacterial pneumonia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001219
- label: Positive Streptococcus pneumoniae sputum culture
- parents: [Sputum infectious agent (HP:0430107)](https://hpo.jax.org/app/browse/term/HP:0430107)
- pmid: PMID:24475349
- def: Streptococcus pneumoniae growth in sputum culture.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10989
- synonyms:
